frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/hooks/useSubscribeCommands/useSubscribeSwitchShowMode.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from 'react'
+import { useUserEditingOrThrow } from '@/stores'
+
+export const useSubscribeShowModeCommand = () => {
+  const { setShowMode } = useUserEditingOrThrow()
+
+  // Switch show mode when ⇧2, ⇧3 or ⇧4 is pressed
+  useEffect(() => {
+    const down = (event: KeyboardEvent) => {
+      if (event.code === 'Digit2' && event.shiftKey) {
+        setShowMode('ALL_FIELDS')
+      }
+      if (event.code === 'Digit3' && event.shiftKey) {
+        setShowMode('TABLE_NAME')
+      }
```

Comment:
https://github.com/user-attachments/assets/a2eb7936-b0bd-4820-b6e1-1860f893631c

Shift+3 doesn't seem to be working. Could you please check?

Reply:
Thanks for your report.
It works on my PC, so the issue may be caused by differences in the environment between us 🤔 
I’d like to clarify whether the problem is caused by ⇧3 or by `setShowMode('TABLE_NAME')`.

I added [some changes for investigation](https://github.com/liam-hq/liam/pull/2683/commits/9c4e145886cc85e26929b03541f8c3897db27f8c) and created another preview -> https://liam-rj464pdmm-liambx.vercel.app

The changes I added are:
- enable ⇧5 to set show mode as "TABLE_NAME", which is the same behaviour with ⇧3
- add console.logs when execute ⇧3 and ⇧5

Could you please try it with new preview and tell me how it works?

https://github.com/user-attachments/assets/0a40001e-73d7-442c-a0ba-2013186d79eb